patternlab/styleguide/source/_patterns/01-atoms/09-media/video.twig

```diff
@@ -1,19 +1,19 @@
 {% set posClass = video.position ? "ma__video--" ~ video.position : '' %}
   {# backward compatibile with v5.6 - toggle new styles with ma__video--new #}
-<div class="ma__video ma__video--new {{ posClass }}" aria-label="Video {{ video.label }}">
+<div class="ma__video ma__video--new {{ posClass }}">
   <div class="ma__video__container js-ma-responsive-video"{% if not video.link %} style="padding-bottom: 13px;"{% endif %}>
     {% if video.link %}
       <div class="ma__video__link">
       {% set video = video|merge({'link': {
                                     'href': video.link.href,
                                     'text': video.link.text,
-                                    'info': video.link.info,
-                                    'labelContext': video.label,
+                                    'info': '',
+                                    'labelContext': video.link.labelContext,
```

Comment:
@joeg8612 --> is this reflected on the drupal side?

Reply:
yes, it looks good.